modules/appnexusBidAdapter.js

```diff
@@ -326,8 +327,8 @@ export const spec = {
     }
   },
 
-  getUserSyncs: function (syncOptions) {
-    if (syncOptions.iframeEnabled) {
+  getUserSyncs: function (syncOptions, responses, gdprConsent) {
```

Comment:
@jsnellbaker This looks good. One question on this piece though. Should this work the same for both client and s2s. It seems to me that the PBS Bid Adapter runs doClientSideSyncs() which in turn calls registerSyncs([]) not passing in any GDPR data when GDPR exists on the bidderRequests. I just want to understand if this is expected or not. The reason being that testing the s2s, the following always returns true since the gdprConsent value is undefined and therefore, the sync iframe runs.

`if (syncOptions.iframeEnabled && hasPurpose1Consent({gdprConsent}))`

Reply:
@mmoschovas Thanks for raising this concern.  

I did some digging and found that the behavior within the `doClientSideSyncs` function wasn't properly working with the gdprEnforcement module.  It was consistently returning this warning: `WARNING: User sync not allowed for null` because we were not properly permitting the bidder to run this code within the special control logic that was added to the `src/config.js` (see `runWithBidder`).

I'm about to push a commit that should address the lack of consent data on this registerSync call and grant the permissions.   Please take a look

CC @snapwich to take a look at the change, to ensure it's formatted properly with the config bidder permission logic (which I believe he helped design).

Reply:
@jsnellbaker your update looks good and it does look like the server -> client syncs call not passing consent data was an oversight (possibly was added before we had standardized on the user sync call). good callout @mmoschovas